content/guides/nextjs/tina-with-strapi/authenticating-with-strapi.md

```diff
@@ -70,9 +70,7 @@ First we'll enable or disable Tina based on whether we're in preview mode.
     apis: {
       strapi: new StrapiClient(process.env.STRAPI_URL),
     },
-    media: {
-      store: new StrapiMediaStore(process.env.STRAPI_URL),
-    },
+    media: store: new StrapiMediaStore(process.env.STRAPI_URL),
```

Comment:
```suggestion
    media: new StrapiMediaStore(process.env.STRAPI_URL),
```

Reply:
Why this declaration is different from the other one?

Based on the documentation, it looks like you can just specify `media`: https://tina.io/docs/media/#adding-a-media-store

See:
https://github.com/tinacms/tinacms/blob/f33c4baf5c8eae59b8edc9c2a7c7ef22482dc38f/packages/%40tinacms/core/src/cms.ts#L130-L132

Reply:
This is the preferred API

Reply:
oh woops I see that `store` wasn't removed. thanks Frank.